global_helpers/panther_event_type_helpers.py

```diff
@@ -1,7 +1,16 @@
 # a place to hold event type constants used among many data models, rules, or policies
 ADMIN_ROLE_ASSIGNED = "admin_role_assigned"
 FAILED_LOGIN = "failed_login"
+LOGOUT = "logout"
```

Comment:
for naming consistency with the `*_LOGIN` would it be worth it to make this `SUCCESSFUL_LOGOUT`? 🤷🏻‍♀️

Reply:
I'm torn here.  I like the consistency but it makes me wonder what an Unsuccessful logout would be

Reply:
I've been in a state where I can't logout from an application before, but it is rare (and the application probably didn't log the fact that is was unsuccessful).  Your call

Reply:
Yeah I think we should do it after pondering it for a few more minutes with a cup of coffee. There could be a situation down the road where we find that log event and better to not have to change every instance of this field